x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_table.tsx

```diff
@@ -129,11 +130,13 @@ const getResponseActionListTableColumns = ({
           <StyledFacetButton
             icon={
               <EuiAvatar
+                title={undefined}
                 name={createdBy}
                 data-test-subj={getTestId('column-user-avatar')}
                 size="s"
               />
             }
+            title={undefined}
```

Comment:
```suggestion
            title="" // Unset EuiFacetButton's default `title` - we're already using an EuiToolTip below
```

Reply:
Thanks @cee-chen. I've made changes as you suggested. Please have a look.